### DISCUSSION TITLE
Possible to define a Left Top 3/4 or 2/3 rectangle

### DISCUSSION THREAD

**Discussion by rastarr** [rxhanson/Rectangle#987] [Q&A]
I was wondering if you could add a Left/Right Top 3/4 or 2/3 rectangle?
On a 1 monitor system with many windows, I find I use these windows for main monitor content with smaller windows in the bottom areas. a 3/4 or 2/3 would be an ideal addition

Or maybe I can configure these myself in an option I haven't found yet

**Comment by rxhanson** [comment]
Corner thirds can be configured via Terminal command:
https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#add-additional-thirds-sizing-commands

To figure out key code & mod flags, you can use an app I made called KeyLimePie (https://keylimepie.app) by opening the shortcut log, or you can use one called KeyCodes that is also on the Mac app store. 

Alternatively, Rectangle Pro (https://rectangleapp.com/pro) has the corner thirds as options in the UI. It does not have 3/4, but that can be configured as a custom shortcut. If you go that route and have trouble getting it set up, just let me know.

**Reply by rastarr** [reply]
I'm a little confused with the wording. I'm looking for, for example, topLeftTwoThird or topLeftThreeFourth
Is that something that can be done via Terminal ?

**Reply by rxhanson** [reply]
I updated that Readme documentation - the commands say "topLeftThird", etc, but the resulting sizes are indeed two-thirds (confusing, I know).

topLeftThreeFourth is not something you can get, and you'd have to either modify the code or create a custom shortcut in [Rectangle Pro](https://rectangleapp.com/pro) for it.

**Reply by rastarr** [reply]
Ahh ok, I was really after the topLeftThreeFourth - thank you anyway for your replies. I know my friend uses Rectangle all the time so I was seeing if it would fit with me. Thanks again
